root/admin/attributes/form.tt

```diff
@@ -34,6 +34,17 @@
         </p>
 
     [% ELSE %]
+        [%~ IF model == "CollectionType" || model == "SeriesType" ~%]
+            [% form_row_select(r, 'entity_type', add_colon(l('Entity type'))) %]
+            [%~ IF r.form.field('entity_type').value.size ~%]
```

Comment:
Also related to errors happening in other fields: this will disable the field even if you're on the "create" form. But that's a minor issue we can probably ignore.